src/Core/Content/Product/Cms/ProductSliderCmsElementResolver.php

```diff
@@ -31,9 +32,16 @@ class ProductSliderCmsElementResolver extends AbstractCmsElementResolver
     private const PRODUCT_SLIDER_ENTITY_FALLBACK = 'product-slider-entity-fallback';
     private const STATIC_SEARCH_KEY = 'product-slider';
     private const FALLBACK_LIMIT = 50;
+    private const PRODUCT_ASSOCIATIONS = [
```

Comment:
'PRODUCT_ASSOCIATIONS' should be deprecated to get removed with Shopware 6.7
See https://github.com/shopware/shopware/issues/5326 & https://github.com/shopware/shopware/pull/5260#issuecomment-2444159644

Reply:
Ah, perfect, yes that is reasonable :-)